TechnicalDocuments/FastlaneMatch.md

```diff
@@ -0,0 +1,121 @@
+Fastlane Match
+==============
+
+Fastlane Match is a tool for managing certificates and provisioning profiles centrally. It downloads certificates and profiles from a GitHub repository or Google Cloud Storage bucket. For safety centrally stored files are encrypted.
+
+Fastlane match [documentation](https://docs.fastlane.tools/actions/match/).
+
+## Several Targets and Types
+
+Best practise for managing different bundle ids is to create one branch for each of them. Default values for the git URL, branch, app identifier and build type are read from the `Matchfile`.
+
+It can happen that code signing certificates for more than one target or type is needed for a build. This can be handled by making several calls to match.
```

Comment:
```suggestion
It can happen that code signing certificates for more than one target or type are needed for a build. This can be handled by making several calls to match.
```